src/assets/main.scss

```diff
@@ -48,6 +48,71 @@ body {
     }
   }
 }
+.control a.is-info {
```

Comment:
why do we need to have all this CSS in main.scss and not in the target component ?